<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterWidget</class>
 <widget class="QWidget" name="FilterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>247</width>
    <height>183</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="allButton">
     <property name="text">
      <string>All</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/images/files5.png</normaloff>:/icons/images/files5.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>ApplyFilter</string>
     </property>
     <property name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/icons/images/ok.png</normaloff>:/icons/images/ok.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
